crypto/signing/mcl/suiteBLS12_381.go

```diff
@@ -22,37 +20,33 @@ type SuiteBLS12 struct {
 	strSuite string
 }
 
-// blsSwapG Is enabled if compiling mcl/bls/bls-go-binary with BLS_SWAP_G flag
+// Is enabled if compiling mcl/bls/bls-go-binary with BLS_SWAP_G flag, curently this is not the case
 // Compiling with the flag will give Public Keys on G1 (48 bytes) and Signatures on G2 (96 bytes)
 // Compiling without the flag will give Public Keys on G2 and Signatures on G1
 // For Elrond the public keys for the validators are known during an epoch and also are not set on blocks
 // BLS signatures are however set on every block header, so in order to optimise the header size flag will be false
 // to have smaller signatures, so on G1(48 bytes)
+const blsSwapG = false
```

Comment:
maybe we should remove the constant as it is no longer used, but keep the comment (adapted) so that the configuration we are using is still clear.

Reply:
done